python/cudf/cudf/_lib/pylibcudf/libcudf/io/types.pxd

```diff
@@ -57,6 +57,18 @@ cdef extern from "cudf/io/types.hpp" \
         ADAPTIVE = 1,
         ALWAYS = 2,
 
+    ctypedef enum column_encoding:
```

Comment:
Rather than a C-style enum, our newer Cython code uses C++ scoped enums, which Cython now supports. [Here's an example](https://github.com/rapidsai/cudf/blob/branch-24.06/python/cudf/cudf/_lib/pylibcudf/libcudf/binaryop.pxd#L14).

Reply:
Thanks for the pointer!